config/s2i/gold/jenkins-continuous-infra-slave-buildconfig-template.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: jenkins-continuous-infra-slave-builder
+metadata:
+  annotations:
+    description: continuous-infra Slaves
+    iconClass: icon-jenkins
+    tags: instant-app,jenkins
+  name: jenkins-continuous-infra-slave-builder
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    annotations:
+      slave-label: fedora-atomic jenkins-continuous-infra-slave continuous-infra-slave
+    labels:
+      role: jenkins-slave
+    name: jenkins-continuous-infra-slave
+  spec: {}
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: jenkins-continuous-infra-slave
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: jenkins-continuous-infra-slave:latest
+    resources: {}
+    source:
+      contextDir: ${CONTEXTDIR}
+      git:
+        ref: ${REPO_REF}
+        uri: ${REPO_URL}
+      type: Git
+    strategy:
+      dockerStrategy:
+      type: Docker
+    triggers:
+    - type: ConfigChange
+parameters:
+- description: Git repository with Dockerfile and slave entrypoint.
+  displayName: Repository URL
+  name: REPO_URL
+  value: https://github.com/CentOS-PaaS-SIG/ci-pipeline.git
+- description: The sub-directory inside the repository.
+  displayName: Context Directory
+  name: CONTEXTDIR
+  value: config/s2i/jenkins/slave
```

Comment:
To echo what Bill said, I believe this should have gold in the path to mirror where this file is being proposed to be added

Reply:
I guess what I am saying is: Nothing in this file is pointing to this current repo so I am not sure why it is being added here without being modified to be hosted in this repo. If the idea is not to have a gold slave image living here, should this file be here?